### ISSUE TITLE
data_base64 and data both set for binary data

### ISSUE THREAD

**Issue by cookingkode** [cloudevents/spec#553]
CloudEventBuilder.build() sets both data_base64 and data in case data is of binary type.  This is a duplication / bloat of payload . Shouldn't they be mutually exclusive?

**Comment by duglin** [comment]
@cookingkode is this for the Java SDK?

**Comment by cookingkode** [comment]
yes.. reference : https://github.com/cloudevents/sdk-java/blob/dfd71734d0d07201a7f1817b8249175be90f979e/api/src/main/java/io/cloudevents/v1/CloudEventBuilder.java#L133 

IMHO just nulling data should work?

**Comment by duglin** [comment]
ok moving to the java sdk repo
